docs/appendix/glossary/api-namespace-triplet.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Api Namespace Triplet
+id: api-namespace-triplet
+full_link:
+short_description: namespace:type:subtype
+aka:
+---
+
+Every Viam {{< glossary_tooltip term_id="resource" text="resource" >}} exposes an [Application Programming Interface (API)](https://en.wikipedia.org/wiki/API) to describe how you can interact with that resource.
```

Comment:
Really, we should add "type" to the glossary and put this in there, and then have the rest of this entry here. Not necessary right now unless you agree and feel so inclined.

Reply:
hmmm-- this is a good catch but now I'm unsure too. @mcvella wrote this section initially so this is not new material, I think it's worthwhile checking in with him-- would both of you consider this change to be more accurate? I guess I'm confused because the models are subtypes, but each subtype has their own resource API wrapper for the SDK, and each one is "exposing" that for communication? However, it is true that the APIs are defined by resource type (we don't use that phrasing a lot elsewhere to refer to types of components and services but that could also be an improvement)

Reply:
agreed with adding type for clarification. Otherwise it does sound a big like a specific resource - like an instantiation of an arm. But I think this means every class/type of resource - like every subtype of arm would expose the same api. So every type sounds right to me.